common/src/main/java/com/wynntils/features/inventory/CustomBankQuickJumpsFeature.java

```diff
@@ -183,6 +172,17 @@ public void onContainerSetEvent(ContainerSetContentEvent.Post e) {
 
             lastPage = currentPage;
         }
+
+        if (!quickJumping) return;
```

Comment:
Are you sure the buttons are set by content, and not set by slot? Which happens earlier? If both happens, using the first would improve speed a bit, using the second packet would improve reliability

Reply:
Just tried it with set slot, it seems to be almost the exact same speed

Reply:
Use the one that is sent later then